exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -120,7 +120,7 @@ drill.exec: {
     affinity.factor: 1.2
   },
   sys.store.provider: {
-    class: "org.apache.drill.exec.store.sys.zk.ZkPStoreProvider",
+    class: "org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider",
```

Comment:
We do that on the perf. cluster :P

In fact, any other _\*PstoreProvider_ has to change _\*PersistentStoreProvider_ after this patch (and reverse if downgrading).